desktop/start.js

```diff
@@ -9,25 +9,35 @@ portfinder.getPortPromise({
     port: basePort,
 }).then((port) => {
     const devServer = `webpack-dev-server --config config/webpack/webpack.dev.js --port ${port} --env platform=desktop`;
-    const buildMain = 'webpack --config config/webpack/webpack.desktop.js --config-name desktop-main --mode=development';
+    const buildMain = 'webpack watch --config config/webpack/webpack.desktop.js --config-name desktop-main --mode=development';
+
+    const env = {
+        PORT: port,
+        NODE_ENV: 'development',
+    };
```

Comment:
It doesn't seem to make a difference whether we pass `env` or don't
I've kept it just in case, but maybe we should just remove it